.github/workflows/maintenance.yaml

```
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: '*/5 * * * *'
+  workflow_dispatch: # Enables on-demand/manual triggering
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - run: |
+          go get github.com/aws/aws-sdk-go
+          go mod tidy
+          go generate ./...
+          cd tools/provider-schema
+          terraform init -upgrade
+          terraform providers schema -json > schema.json
+      - uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: |
+            autogenerated maintenance
+          title: autogenerated maintenance
```

```suggestion
          title: Generate rules from terraform-provider-aws
```

Would be nice to append `@version` as well, but that may increase the difficulty of targeting the PR. Dependabot/Renovate do something like this but probably w/ considerable logic involved.

---

also not elegant, but I've just manually appended version numbers to autogenerated PR titles in the past due to complexities of figuring it out automatically:
https://github.com/aws-cloudformation/cfn-lint-visual-studio-code/pull/76